protocol/contracts/beanstalk/metadata/MetadataFacet.sol

```diff
@@ -5,8 +5,10 @@
 pragma solidity ^0.7.6;
 pragma experimental ABIEncoderV2;
 
-import {LibLegacyTokenSilo} from "contracts/libraries/Silo/LibLegacyTokenSilo.sol";
 import "./MetadataImage.sol";
+import "contracts/mocks/MockERC1155.sol";
```

Comment:
Why is a Mock contract imported in a main facet? Should this be removed?

Reply:
left over from testing, fixed: 9440bbb5c92bdd59b91e39e950699e3a0baaa4f8